flask-connexion-rest-part-2/version_1/models.py

```diff
@@ -12,7 +12,15 @@ class Person(db.Model):
     )
 
 
-class PersonSchema(ma.ModelSchema):
+# flask-marshmallow<0.12.0
+
+# class PersonSchema(ma.ModelSchema):
+#    class Meta:
+#        model = Person
+#        sqla_session = db.session
+
+# flask-marshmallow>=0.12.0 (recommended)
+class AuthorSchema(ma.SQLAlchemyAutoSchema):
     class Meta:
         model = Person
         sqla_session = db.session
```

Comment:
Is `sqla_session` still needed?

Reply:
Actually no, it is not needed. I just pushed a fix that leaves the example working.